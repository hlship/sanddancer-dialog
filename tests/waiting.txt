%% Nothing in emotional baggage, choose at random
z
z
brood grandma
%% Choose randomly from what's been brooded about:
z
z