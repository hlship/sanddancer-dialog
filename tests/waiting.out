

Middle of Nowhere (in the pickup truck)
You've smashed up against a tall Saguaro, which is all you can see out the
windshield except a few feet of desert sand, then blackness. Hell, you must have
driven off the road and crashed. Ocean keeps telling you this night commute on
this lonely highway's gonna kill you; maybe you should start listening.

On the dash you can see a piece of jade.

The glove compartment is closed.

Through the windshield, you see a little lizard clinging frantically to the
Saguaro.

> z
Nah, man, better keep busy or else it's back to brooding about your shit job.

Something about the piece of jade makes you want to brood about your grandma's
stories.

> z
Nah, man, better keep busy or else it's back to brooding about your grandma's
stories.

> brood grandma
There are dark spirits who roam the earth, little Knock. Grandma used to say
that, holding you tight and stroking your hair. There are dark spirits who roam
the earth, but you're not alone. Oh, no. I'm here. (She'd kiss your head and
you'd squeeze her back.) But others are watching out for you too. You have three
animal guardians, hmm? Spirits who are always watching over you. Oh, you can't
always trust them to know what's best. Remember that, Knock. But when you need
help, they'll come, and protect you from the worser things in the world.

Mom would yell at Grandma a lot for filling your head with that new-age
bullshit. Grandma grew up white and midwestern and Baptist, but had started
wearing things with feathers and playing the pan flute by the time your were
born. She seemed to really like having a son-in-law who was Native American or
American Indian or Indigenous People or whatever she'd decided the term was that
week, and she was pretty pissed when Mom left him. Anyway. Her stories were
mostly BS, you guess, but some of them stuck with you. When it's dark you still
wonder if your spirit animals are out there, somewhere, and what the hell is
taking them so long to find you.

> z
Nah, man, better keep busy or else it's back to brooding about your grandma's
stories.

> z
Nah, man, better keep busy or else it's back to brooding about your grandma's
stories.

> 
